includes/classes/PullListTable.php

```diff
@@ -222,35 +225,37 @@ public function column_date( $post ) {
 	/**
 	 * Output standard table columns (not name)
 	 *
-	 * @param  array  $item Item to output.
-	 * @param  string $column_name Column name.
+	 * @param  array|\WP_Post $item Item to output.
+	 * @param  string         $column_name Column name.
+	 *
+	 * @return string Url, post title, or empty string.
 	 * @since  0.8
 	 */
 	public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 			case 'name':
-				return $item['post_title'];
-				break;
+				return $item['post_title']; // return is just as good as break
```

Comment:
Change looks good, I'd remove the comment though, don't think it adds much here.